src/esp/assets/ResourceManager.cpp

```diff
@@ -649,6 +687,113 @@ Magnum::Range3D ResourceManager::computeMeshBB(BaseMesh* meshDataGL) {
   return Magnum::Range3D{
       Magnum::Math::minmax<Magnum::Vector3>(meshData.positions)};
 }
+void ResourceManager::computePTexMeshAbsoluteAABBs(BaseMesh& baseMesh) {
```

Comment:
IMO, it makes sense to `#ifdef` the function definition itself instead of the implementation.  If we aren't building with ptex, then we can assume this method should/will never be called, so there shouldn't be any code that requires this blank implementation right?

Reply:
Makes sense to me. Will do.

Reply:
Still #ifdef'ing the implementation and not the whole function.

Reply:
I actually cannot add `#ifdef` to the entire function definition since it would cause the CI test failure. When compiling js, it complained undefined symbols. 

If you look at the other functions, e.g., `loadPTexMeshData`, we did the same thing, namely, adding the conditionals on the body, not the entire definition. So the function is just empty.

I guess I will just leave it like this.

@msbaines  @erikwijmans

Reply:
I think you also need to `#ifdef` it in the header file.